stix_edh/bindings/cyber_profile.py

```diff
@@ -817,9 +823,10 @@ def buildAttributes(self, node, attrs, already_processed):
         super(FurtherSharingType, self).buildAttributes(node, attrs, already_processed)
     def buildChildren(self, child_, node, nodeName_, fromsubclass_=False):
         if nodeName_ == 'sharingScope':
-            sharingScope_ = child_.text
-            sharingScope_ = self.gds_validate_string(sharingScope_, node, 'sharingScope')
-            self.sharingScope = sharingScope_
+            self.gds_validate_string(child_.text, node, 'sharingScope')
```

Comment:
The previous version of the code saved the result to a variable and used it. Is it okay that we aren't doing that?

Reply:
Actually, the generated code for this didn't work.  I'm not sure why - so I "stole" this code from AccessPrivilegeType, which has a similar property to sharingScope (see Line 724)

Reply:
sharingScope_ = child_.text
            sharingScope_ = self.gds_validate_string(sharingScope_, node, 'sharingScope')
            self.sharingScope.append(sharingScope_)